garage/tf/algos/npo.py

```diff
@@ -390,7 +390,7 @@ def _build_entropy_term(self, i):
                 policy_dist_info = self.policy.dist_info_sym(
                     i.obs_var,
                     i.policy_state_info_vars,
-                    name='policy_dist_info')
+                    name='policy_dist_info_entropy')
```

Comment:
I renamed it because we can't call `dist_info_sym` with the same name twice (because the underlying model will build twice with the same name, which is not allowed). I think Chang is going to fix it by changing the `name_scope` to `variable_scope`. @CatherineSue

Reply:
Changing it from `name_scope` to `variable_scope` won't fix the error. The cause is in Model instead of here.